certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "1.15.0",
+		"title": "RAPID Platform (V1)",
+		"description": "The RAPID Platform connector allows you to interact with your RAPID sites to help automate your business.",
```

Comment:
Could this description be expanded a little bit? Just another sentence or two talking about the value of the connector/functionality it offers would be great.

Reply:
Have updated with 3rd pass from marketing with an expanded description while trying to match the language style of other published connectors.